packages/core/src/oidc/init.ts

```diff
@@ -40,11 +40,12 @@ export default async function initOidc(app: Koa): Promise<Provider> {
       keys,
     },
     features: {
-      userinfo: { enabled: true },
+      userinfo: { enabled: false },
       revocation: { enabled: true },
       devInteractions: { enabled: false },
       resourceIndicators: {
         enabled: true,
+        defaultResource: () => '',
```

Comment:
Maybe better the reference

```suggestion
        // https://github.com/panva/node-oidc-provider/blob/main/docs/README.md#defaultresource
        defaultResource: () => '',
```

Reply:
found this cannot be located when summary is collapsed. updated to a general id `resourceindicators`